### PR TITLE
bug & typo fixes for #373

### PR DESCRIPTION
**Pull request by chris18890**
replace mysql_ for mysqli_ & add variable #373 #366 #346

### CODE REVIEW

**Comment by chris18890** [comment]
@elplatt please merge when you get the chance

File: crm/modules/amazon_payment/amazon_payment.inc.php [hunk 1]
```diff
@@ -481,7 +484,7 @@ function amazon_payment_contact_delete_form ($cid) {
  * @param &$form_data Metadata about the form.
  * @param $form_id The name of the form.
  */
-function amazon_payment_form_alter($form, $form_id) {
+function amazon_payment_form_alter ($form, $form_id) {
```
Comment (ramgarden) [inline crm/modules/amazon_payment/amazon_payment.inc.php]: Why was this space added between function name and parameter list?

File: crm/modules/paypal_payment/paypal_payment.inc.php [hunk 2]
```diff
@@ -306,13 +306,13 @@ function paypal_payment_payment_api ($payment, $op) {
...
- * Generate payments contacts table
+ * Generate payments contacts table.
  *
  * @param $opts an array of options passed to the paypal_payment_contact_data function
  * @return a table (array) listing the contacts represented by all payments
  *   and their associated paypal email
- */ 
-function paypal_payment_contact_table($opts){
+ */
+function paypal_payment_contact_table ($opts) {
```
Comment (ramgarden) [inline crm/modules/paypal_payment/paypal_payment.inc.php]: Spaces around the function parameter list?  I guess I've missed some update in the coding standard?

Comment (chris18890) [inline crm/modules/paypal_payment/paypal_payment.inc.php]: just comparing it to other functions in the same file that seem to have spaces before & after it

File: crm/modules/amazon_payment/amazon_payment.inc.php (hunk 1 above)
Comment (chris18890) [inline crm/modules/amazon_payment/amazon_payment.inc.php]: just comparing it to other functions in the same file that seem to have spaces before & after it

**Pull request merged**